messages/store.json

```
@@ -127,21 +127,24 @@
             "extensionPointNameDescription": "Means of identifying entry points within the domain code that are capable of being extended by partners.Takes the form domain_sub-domain_interface. E.g: checkout_summary_computeTax",
             "regExtensionNameFlagDescription": "Unique name for extension",
             "errEPN": "Invalid EPN. Run [commerce:extension:points:list] for all valid EPN values",
-            "savingConfigIntoConfig": "Sucessfully registered the extension. Use command [commerce:extension:map] to link the extension to a store.",
+            "savingConfigIntoConfig": "Successfully registered the extension. Use command [commerce:extension:map] to link the extension to a store.",
             "apexNameSpaceFlagDescription": "Optional namespace for apex class",
             "undefinedName": "Undefined registered extension name. Enter '-r <name>'"
         },
         "map": {
             "cmdDescription": "Maps an extension to a specific webstore",
-            "StoreNameFlagDescription": "The name of the web store to map the extension",
-            "errStoreName": "Web-storename is not valid",
-            "errStoreId": "Store Id is not valid",
-            "nameAlreadyExists": "Registered extension name already exists: ",
+            "StoreNameFlagDescription": "The name of the webstore to map the extension",
+            "errStoreName": "Can't find webstore with the name:",
+            "errStoreId": "Can't find store ID:",
```

we should use either `webstore` or `store`  in all the messages to keep it consistent
```suggestion
            "errStoreId": "Can't find webstore with the ID:",
```